src/rns/hooks/sold-domain.hooks.ts

```diff
@@ -18,38 +19,41 @@ export default {
         if (!context.params.query) {
           context.params.query = {}
         }
-
-        if (context.params.route?.ownerAddress) {
-          context.params.query.sellerAddress = context.params.route.ownerAddress.toLowerCase()
-        }
       }
     ],
     find: [
       (context: HookContext) => {
-        context.params.sequelize = {
-          raw: false,
-          nest: true,
-          include: {
-            model: Domain
-          }
-        }
-        const { params: { sequelize: { include } } } = context
         const { domain } = context.params.query as any
+        let nameSearch
 
-        if (include && domain) {
-          const { name: { $like } } = domain
-          include.where = {
+        if (domain) {
```

Comment:
As we discussed, please change the queried name to be retrieved as `const { name: { $like } } = domain` so that the structure of the UI query is consistent with what the UI gets from the endpoint.
Thank you:)

Reply:
Done! and you have a gift https://github.com/rsksmart/rif-marketplace-ui/pull/138